ha/core/system_health/bootstrap/json_kv_convertor.py

```diff
@@ -43,85 +52,156 @@ def __init__(self):
 
 
     # collct kvs and if all 3 values for a component are avilable, generate healthEvent
-    def _update_health(self, key, val):
-
+    def _update_health(self, key: str, val: str) -> None:
```

Comment:
Please update doc string and function as per python standard shared in channel

Reply:
Done.